CDCgov/NEDSS-Modernization
# adding configuration for search criteria to be autofilled in add patient page

This Pr covers this use case AS A Data Entry and Search User
-- I WANT TO have my search results automatically filtered into a new patient when I create a new patient after a search
-- SO THAT I Can reduce repetitive data entry after completing a search

## Description
adding configuration for search criteria to be autofilled in add patient page 


## Tickets

* [Jira Ticket](url)
https://cdc-nbs.atlassian.net/browse/CNFT1-1466

## Steps to verify

1. Enter information into any input field for regular search
2. After search click on add patient 
3. See the searched criteria's being autofilled into the add patient page
